tests/schema/valid.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "../../deeployer.schema.json",
+    "containers": {
+        "elasticsearch": {
+            "image": "",
+            "replicas": "1",
+            "ports": "",
+            
+
+            "env" : {
+                "FOO": "BAR"
+            },
+
+
+            "host": "",
+
+            "volumes" : {},
+
+
+            "quotas" : {
+
+                "min": {
+                    "cpu": "",
+                    "memory" : ""
```

Comment:
For most of the string values put in the JSonSchema, we might want to say they cannot be empty.

For instance, "cpu" can be unspecified, but should not be an empty string.
We can check that in JsonSchema using:

```
{
  "type": "string",
  "minLength": 1
}
```